pom.xml

```diff
@@ -3,27 +3,48 @@
                       http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <groupId>org.neo4j.driver</groupId>
+  <artifactId>neo4j-java-driver-parent</artifactId>
+  <version>4.2-SNAPSHOT</version>
+
+  <packaging>pom</packaging>
+  <name>Neo4j Java Driver Project</name>
+  <description>A project for building a Java driver for the Neo4j Bolt protocol.</description>
+  <url>https://github.com/neo4j/neo4j-java-driver</url>
+
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
     <rootDir>${project.basedir}</rootDir>
-    <surefire.and.failsafe.version>2.22.1</surefire.and.failsafe.version>
-    <junit.version>5.7.2</junit.version>
     <parallelizable.it.forkCount>1C</parallelizable.it.forkCount>
     <!-- All tests tagged are to be executed in parallel -->
     <parallelizable.it.tags>parallelizableIT</parallelizable.it.tags>
-  </properties>
-
-  <groupId>org.neo4j.driver</groupId>
-  <artifactId>neo4j-java-driver-parent</artifactId>
-  <version>4.2-SNAPSHOT</version>
+    <surefire.and.failsafe.version>2.22.1</surefire.and.failsafe.version>
```

Comment:
TIL they seem to always be released together (since 2.5, the first version of failsafe)